src/Raven.Server/Documents/PeriodicBackup/Restore/AzureRestorePoints.cs

```diff
@@ -45,7 +45,7 @@ protected override async Task<List<FileInfoDetails>> GetFiles(string path)
             foreach (var obj in allObjects.ListBlob)
             {
                 if (TryExtractDateFromFileName(obj.Name, out var lastModified) == false)
-                    lastModified = Convert.ToDateTime(obj.LastModified);
+                    continue;
```

Comment:
Why we changed this?

Reply:
we can skip it, it is not a backup file.